csrc/group_gemm_groupwise_sm100.cu

```diff
@@ -57,6 +57,7 @@ void CutlassGroupGemmGroupwiseScaledSM100(at::Tensor int_workspace_buffer,
   const c10::cuda::OptionalCUDAGuard device_guard(float_workspace_buffer.device());
   auto stream = at::cuda::getCurrentCUDAStream();
   int batch_size = m_indptr.size(0) - 1;
+  int max_m = SFA.size(1);
```

Comment:
Is it compatible with CUDAGraph?
When used together with CUDAGraph, this value will only be evaluated once (in the capture stage), and reused in all later replay runs. Can we guarantee the `max_m` inferred in capture stage is compatible with the value used in replay stage?

Reply:
Per discussion w/ @cyx-6 . We assume SFA have already been padded to its maximum size when used in CUDAGraph mode. This assumption should be articulated in the documentation.